src/cachingreader.cpp

```diff
@@ -246,40 +267,20 @@ void CachingReader::process() {
                 qDebug() << "ERROR: status.chunk is NULL in CHUNK_READ_SUCCESS ReaderStatusUpdate. Ignoring update.";
                 continue;
```

Comment:
When can this happen? Is continue here the right error recover strategy? 
A comment could be helpfull.